tensorflow_addons/image/resampler_ops_test.py

```diff
@@ -0,0 +1,253 @@
+# pylint: disable=bad-continuation
```

Comment:
This will disable it on the whole file. Maybe you just want to disable it for a block?
```python
# pylint: disable=bad-continuation
DO SOMETHING
# pylint: enable=bad-continuation
```

Reply:
Thanks for pointing this out, @WindQAQ.